test/cypress/integration/dashboard/dashboard-links.cy.js

```diff
@@ -66,8 +66,25 @@ describe('the links are displayed correctly on the dashboard', () => {
       cy.get('#payment-links-link').should('have.class', 'flex-grid--column-half')
     })
 
-    it('should display 4 links for a test sandbox account', () => {
-      cy.task('setupStubs', getStubsForDashboard(gatewayAccountId, 'test', 'sandbox', 'NOT_STARTED'))
+    it('should display 3 links for a test sandbox account created since onboarding flow changed on 29/08/2024', () => {
+      cy.task('setupStubs', getStubsForDashboard(gatewayAccountId, 'test', 'sandbox', 'NOT_STARTED', null, '2024-08-30'))
+
+      cy.visit(dashboardUrl)
+      cy.get('.links__box').should('have.length', 3)
+
+      cy.get('#demo-payment-link').should('exist')
+      cy.get('#demo-payment-link').should('have.class', 'flex-grid--column-third')
+
+      cy.get('#test-payment-link-link').should('exist')
+      cy.get('#test-payment-link-link').should('have.class', 'flex-grid--column-third')
+
+      cy.get('#request-to-go-live-link').should('exist')
+      cy.get('#request-to-go-live-link').should('have.class', 'flex-grid--column-third')
+
```

Comment:
Think it'd be good to test for the non-existence of `#request-stripe-test-account` too

Reply:
Good point. Updated.